client/cli/src/main.rs

```diff
@@ -42,13 +42,13 @@ enum Command {
     /// Reissue tokens received from a third party to avoid double spends
     Reissue {
         #[clap(parse(from_str = parse_coins))]
-        coins: Coins<SpendableCoin>,
+        coins: TieredMulti<SpendableNote>,
```

Comment:
What about `TieredVec`?  `Multi` is a bit ambiguous.

Guess the parameter names should be changed, were you going to do that as well?

Reply:
I took the `TieredMulti` from the existing `TieredMultiZip`. I'm not too happy about this name and slightly prefer `TieredVec` as well. But it's also not ideal. :D

Reply:
Another option is `Denomination` which is even more clear in the context of bank `Note`, but maybe too long to type...

Will you also change the parameter name e.g. `coins:` to something like `notes` or `ecash`?

Reply:
I have a follow up PR with more renames, etc.

This enum variant could be just a newtype and there would be no need to name anything.